devops/create_https_wss_endpoints_aws/creation_certificates.py

```diff
@@ -2,30 +2,56 @@
 import boto3
 from collections import defaultdict
 
-
 dict_region_sslcerts = defaultdict(list)
 
-
 pp = pprint.PrettyPrinter(indent=4)
 
+
 # step 1: request SSL certificates
-def request_ssl_certificates(region, dn):
+def request_ssl_certificates(region, dn, dict_exist_sslcerts):
     """
     Notes:
         * idempotent ops
         * store CertificateArn to dict_region_sslcerts
     """
 
     print("\n==== step 1: request SSL certificates, CertificateArn will be stored into dict_region_sslcerts \n")
+    # skip if certificate already exist
+    if dn in dict_exist_sslcerts:
```

Comment:
If we passed the domain name in the previous method, the returned dict would be empty if no matching pattern found.